src/semgrep_agent/targets.py

```diff
@@ -285,11 +285,18 @@ def baseline_paths(self) -> Iterator[List[Path]]:
         :return: A list of paths
         :raises ActionFailure: If git cannot detect a HEAD commit or unmerged files exist
         """
-        if self._base_commit is None:
+        repo = get_git_repo()
```

Comment:
Probably worth mentioning in the function comment that this returns empty list if not a git repo and otherwise returns elements in target paths that were not in added

Reply:
good call! added